README.md

```diff
@@ -456,12 +456,10 @@ Housekeeping is the process that removes expired and orphaned data on the replic
 
 |Property|Required|Description|
 |----|----|----|
-|`housekeeping.h2.database`|No|The location of the h2 housekeeping database. Defaults to `${instance.home}/data/${instance.name}/housekeeping`.|
-|`housekeeping.expired-path-duration`|No|Time To Live (TTL) of legacy replica paths in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) format: only days, hours, minutes and seconds can be specified in the expression.|
-|`housekeeping.data-source.driver-class-name`|No|Java classname of the database JDBC driver.|
-|`housekeeping.data-source.url`|No|JDBC connection URL. Refer to the [Instance Globals](#instance-globals) section for more details.|
-|`housekeeping.data-source.username`|No|database user with access to `circus_train` schema.|
-|`housekeeping.data-source.password`|No|database user's password.|
+|`housekeeping.h2.database`|No|The location of the h2 housekeeping database. Defaults to `${instance.home}/data/${instance.name}/housekeeping`. Refer to the [Instance Globals](#instance-globals) section for more details.|
```

Comment:
This is Circus Train specific and isn't actually in Housekeeping?

Reply:
oh it's a bit complex:
`housekeeping.h2.database` is in housekeeping But the reference to the instance globals isn't. I don't really like that in housekeeping tbh, but in housekeeping we set this as well. Perhaps I just copy that, I'll sleep on this

Reply:
I've added an entry on this in the HK readme as well, keeping this here as it makes more sense in this context imo

Reply:
OK, so we keep the ones most often used by CT duplicated here and then provide a link to the rest for those who might want to customise it further? I'm OK with this, let's just try get the text to be the same (i.e. copy over from HK PR once that ones merged).

Reply:
No the text shouldn't be the same because in CT context `instance.home` for instance has a default value (`user.home`). I think it makes more sense leaving like this reading it as a CT user???

Reply:
OK, yeah, that's fine.